linkerd/app/inbound/src/accept.rs

```diff
@@ -46,6 +49,19 @@ impl<N> Inbound<N> {
     {
         self.map_stack(|cfg, rt, accept| {
             accept
+                .push_on_service(svc::MapErr::layer(Into::into as fn(_) -> Error))
+                .push_map_target(|(policy, t): (AllowPolicy, T)| {
+                    tracing::debug!(policy = ?&*policy.borrow(), "Accepted");
+                    Accept {
+                        client_addr: t.param(),
+                        orig_dst_addr: t.param(),
+                        policy,
+                    }
+                })
+                .push(policy::Discover::layer(policies))
+                .push(svc::MapErr::layer(Into::into as fn(_) -> Error))
+                .into_new_service()
```

Comment:
How does backpressure work on this service? Is this discovery cached/reused?

Reply:
`policies` here is a `Store`, which is (still) an idle cache of resolutions. `Store` will still still cache and reuse the discovery the way it did previously; the only difference in `Store`'s behavior is that we now return a future that waits for the _initial_ watch to start, rather than returing the default policy if the watch has not yet started: https://github.com/linkerd/linkerd2-proxy/blob/d64d41639649f3d6ec27b550e4099120cb064ae5/linkerd/app/inbound/src/policy/store.rs#L121-L154

The `Discover` service doesn't advertise meaningful backpressure because the source of backpressure would be the tonic client, and tonic clients don't advertise backpressure...